invenio_app_rdm/records_ui/views/deposits.py

```diff
@@ -471,11 +472,12 @@ def deposit_edit(pid_value, draft=None, draft_files=None, files_locked=True):
         files_locked=files_locked,
         permissions=draft.has_permissions_to(
             [
-                "manage",
-                "new_version",
                 "delete_draft",
+                "manage",
                 "manage_files",
                 "manage_record_access",
+                "new_version",
+                "publish",
```

Comment:
this seems to be the only new value, the rest is just reordered?

Reply:
Exactly. Reordered them alphabetically and added `publish`